readme.md

```diff
@@ -54,12 +55,14 @@ The effective limits will then be a calculated value from `MAX_SCRIPT_ELEMENT_SI
 rather than independently set limits (as they were before this upgrade).
 This is equivalent because script numbers are LE integers with highest bit used as sign bit so a stack item of `MAX_SCRIPT_ELEMENT_SIZE` size and filled with all `0xff` bytes will be equal to `MIN_SCRIPTNUM`, and the same item with highest bit flipped (`0xffff...ff...ff7f`) will be equal to `MAX_SCRIPTNUM`.
 
-The [`CHIP: Targeted Virtual Machine Limits`](https://github.com/bitjson/bch-vm-limits) will set `MAX_SCRIPT_ELEMENT_SIZE` to 10,000 bytes, therefore valid range for results of arithmetic operations shall be the inclusive range: `[-2^79999 + 1, 2^79999 - 1]`.
-
 Any operation that would result in a stack item bigger than `MAX_SCRIPT_ELEMENT_SIZE` must fail, e.g. `{stack: a} OP_1ADD` must fail for `a = MAX_SCRIPTNUM`.
 
 By coupling the numerical limit with `MAX_SCRIPT_ELEMENT_SIZE` we can guarantee that **any stack item** can be a valid input to an arithmetic operation, e.g. `{stack: a} OP_BIN2NUM OP_DUP OP_NUMEQUAL` will pass for any `a` on stack, since it will not be possible to have a stack item that would be interpreted as value outside the `[MIN_SCRIPTNUM, MAX_SCRIPTNUM]` range.
 
+The [`CHIP: Targeted Virtual Machine Limits`](https://github.com/bitjson/bch-vm-limits) will set `MAX_SCRIPT_ELEMENT_SIZE` to 10,000 bytes, therefore valid range for results of arithmetic operations will effectively be the inclusive range: `[-2^79999 + 1, 2^79999 - 1]`.
```

Comment:
As mentioned in the main PR threads, this assertion of the range deserves a derivation to be convincing. It's not yet crystal clear that it's the smallest and largest number that can fit in a max byte size VM number.

Reply:
umm it's demonstrated just above, 10k `0xff` bytes is same value as 2^79999-1

Reply:
Right. If it were me, I would show the intermediate step

`[-2^(10000 * 8 - 1) + 1, 2^(10000 * 8 -1) - 1]` (I didn't double check that)

I consider it fine either way after the equation is fixed.

Reply:
ah sure, yeah we can show that